.github/packaging/SharpVector.nuspec

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<package >
+    <metadata>
+        <id>SharpVector</id>
+        <version>1.0.0</version>
+        <description>
+            SharpVector utility - std::vector with all .NET's List features: ForEach, FindAll, and more!

```

Comment:
Due to the project renaming this will have to be updated:
*SharpVector* -> *ExtendedVector*